src/api/participants/handler.js

```diff
@@ -90,21 +90,22 @@ const create = async function (request, h) {
     }
     const ledgerAccountIds = Util.transpose(ledgerAccountTypes)
     const allSettlementModels = await SettlementService.getAll()
-    const settlementModels = allSettlementModels.filter(model => model.currencyId === request.payload.currency)
-    if (Array.isArray(settlementModels) && settlementModels.length > 0) {
+    let settlementModels = allSettlementModels.filter(model => model.currencyId === request.payload.currency)
```

Comment:
This is much more readable! Thanks.

Maybe I misunderstood the original purpose, but it looks like if there are settlement models found on line `93` we never do anything with them (e.g. line 99). Is there meant to be another  closed bracket on line `99`?

Reply:
ah you are right!